lib/ansible/modules/notification/irc.py

```diff
@@ -133,8 +133,14 @@
 import time
 import traceback
 
+from ansible.module_utils._text import to_native, to_text, to_bytes
 from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils._text import to_native
+from ansible.module_utils.six import PY3
+
+if PY3:
+    safe = to_bytes
+else:
+    safe = to_text
```

Comment:
This is the same as to_native so it's not needed

Reply:
no, its the reverse